cxx-squid/src/main/java/org/sonar/cxx/CxxAstScanner.java

```diff
@@ -124,7 +125,18 @@ public SourceCode createSourceCode(SourceCode parentSourceCode, AstNode astNode)
           sb.append(token.getValue());
         }
         String functionName = sb.toString();
-        SourceFunction function = new SourceFunction(functionName + ":" + astNode.getToken().getLine());
```

Comment:
This part is creating the qualified name including the namespaces. Going from current node upwards the tree adding all namespace names.